packages/components/inputs/rich-text-input/src/rich-text-input.visualroute.jsx

```diff
@@ -30,7 +30,7 @@ const InteractiveRoute = () => {
 
   return (
     <Suite>
-      <Spec label="minimal" omitPropsList>
+      <Spec label="minimal" omitPropsList testedThemes={['new']}>
```

Comment:
Hey @commercetools/shield-team-fe 👋

Let me ask one more question before I invest more time into that.
We have some interactive Puppeteer tests where series of actions run against a component. This applies to components:
- [`<AsyncCreatableSelectInput>`](https://github.com/commercetools/ui-kit/blob/cb0a756478ff7a54c3ffaa17c4d3329d1c6fa3aa/packages/components/inputs/async-creatable-select-input/src/async-creatable-select-input.visualspec.js#L11-L20)
- [`<AsyncSelectInput>`](https://github.com/commercetools/ui-kit/blob/cb0a756478ff7a54c3ffaa17c4d3329d1c6fa3aa/packages/components/inputs/async-select-input/src/async-select-input.visualspec.js#L11-L20)
- [`<DateInput>`](https://github.com/commercetools/ui-kit/blob/cb0a756478ff7a54c3ffaa17c4d3329d1c6fa3aa/packages/components/inputs/date-input/src/date-input.visualspec.js#L10-L35)
- [`<LocalizedRichTextInput>`](https://github.com/commercetools/ui-kit/blob/cb0a756478ff7a54c3ffaa17c4d3329d1c6fa3aa/packages/components/inputs/localized-rich-text-input/src/localized-rich-text-input.visualspec.js#L55-L145)
- [`<RichTextInput>`](https://github.com/commercetools/ui-kit/blob/cb0a756478ff7a54c3ffaa17c4d3329d1c6fa3aa/packages/components/inputs/rich-text-input/src/rich-text-input.visualspec.js#L55-L370)
- [`<SearchSelectInput>`](https://github.com/commercetools/ui-kit/blob/cb0a756478ff7a54c3ffaa17c4d3329d1c6fa3aa/packages/components/inputs/search-select-input/src/search-select-input.visualspec.js#L11-L20)

Those tests were intended to run against a single component existing in a document (they use isolated paths and query functions that are used in those tests expect only one piece of tested component in the document). Therefore, if we render the component in both themes in a document - the tests break without a refactor.

The [snapshots are already disabled](https://github.com/commercetools/ui-kit/blob/cb0a756478ff7a54c3ffaa17c4d3329d1c6fa3aa/packages/components/inputs/async-creatable-select-input/src/async-creatable-select-input.visualspec.js#L18-L19) in all of those tests due to issues with `percy`. 

Would you say we need a run-through of all user actions for both themes or using the new theme is enough for us?

Reply:
> Would you say we need a run-through of all user actions for both themes or using the new theme is enough for us?

The new theme only is enough for me.